containers/Investment/JuniorRatio/index.tsx

```diff
@@ -61,19 +61,19 @@ class JuniorRatio extends React.Component<Props, State> {
     const { minJuniorRatio } = this.state;
     return <Box pad={{ horizontal: 'medium' }}>
             <Box direction="row" margin={{ top: 'medium' }}>
-                <Heading level="4">Set minimum TIN ratio</Heading>
+                <Heading level="4">Set minimum TIN/DROP ratio</Heading>
             </Box>
             <Box direction="row" gap="medium" >
                 <Box basis={'1/3'}>
-                    <FormField label="Min TIN ratio">
+                    <FormField label="Min TIN/DROP ratio">
                         <NumberInput value={baseToDisplay(minJuniorRatio, 27)} precision={2}
                             onValueChange={({ value }) =>
                                 this.setState({ minJuniorRatio: displayToBase(value, 27) })}
                         />
                     </FormField>
                 </Box>
                 <Box align="start">
-                    <Button primary label="Set min TIN ratio" onClick={this.setMinJuniorRatio} />
+                    <Button primary label="Set min TIN/DROP ratio" onClick={this.setMinJuniorRatio} />
                 </Box>
```

Comment:
I think it should be: Set min TIN ratio 🙈

Reply:
Okay, reset :)